ext/gd/gd.stub.php

```diff
@@ -224,6 +224,12 @@ function imagettfbbox(float $size, float $angle, string $font_file, string $text
 function imagettftext(GdImage $im, float $size, float $angle, int $x, int $y, int $col, string $font_file, string $text): array|false {}
 #endif
 
+/**
+ * @param int|float $arg1
```

Comment:
Can't this be used in the declaration? There is a ZPP Check for ``int|float`` already

Reply:
No unfortunately. And no sane person would touch the ZPP of this function. This is probably one of the worst ZPP handling I've ever seen.

Reply:
Ok, my last comment was a exaggerating IMO, but nevertheless, the solution used in `imagefilter()` feels a bit hacky :D

Reply:
I think it would be more honest to make this signature `imagefilter(GdImage $im, int $filtertype, ...$filterArgs)`.

Reply:
You're right! And I've just fixed it :)